### DISCUSSION TITLE
failed creating pyroscope: compactor block range periods should be divisible by the max block duration, but 1h0m0s is not divisible by 3h0m0s

### DISCUSSION THREAD

**Discussion by Raghav65** [grafana/pyroscope#2764] [Q&A]
I'm using grafana/pyroscope version v1.2.0

I have configured compactor block_ranges as default and I'm getting below error message as by default block_ranges: <list of durations> | = 1h0m0s,2h0m0s,8h0m0s. 

failed creating pyroscope: compactor block range periods should be divisible by the max block duration, but 1h0m0s is not divisible by 3h0m0s

When I try to add below compactor config in compactor block in config.yaml I'm getting - " failed parsing config: /etc/pyroscope/config.yaml: yaml: unmarshal errors:   line x: cannot unmarshal !!str `3h0m0s` into compactor.DurationList"

  -compactor.block-ranges value
    	List of compaction time ranges. (default 3h0m0s,6h0m0s,12h0m0s)

    compactor:
        block_ranges: 3h0m0s

max_block_duration is set to 3h0m0s

**Comment by cyriltovena** [comment]
Fixes here https://github.com/grafana/pyroscope/issues/2768
